mysqld_exporter.go

```diff
@@ -205,7 +205,7 @@ var landingPage = []byte(`<html>
 // Metric descriptors for dynamically created metrics.
 var (
 	binlogSizeDesc = prometheus.NewDesc(
-		prometheus.BuildFQName(namespace, binlog, "size"),
+		prometheus.BuildFQName(namespace, binlog, "size_bytes"),
```

Comment:
Actually, I'm not even sure if this is in bytes or not, which is exactly the problem - please verify.